README.md

```diff
@@ -206,6 +206,25 @@ Quagga2 exports the BarcodeReader prototype, which should also allow you to crea
 reader implementations using the base BarcodeReader implementation inside Quagga2.  The QR reader
 does not make use of this functionality, as QR is not picked up as a barcode in BarcodeReader.
 
+### External Reader Priority
+
```

Comment:
should there be similar documentation somewhere in the /docs configuration.md ?
also, is there /docs documentation about external readers?  that should be filled out, possibly in tutorials, howto, and reference sections?